test/integration/dockerregistryclient_test.go

```diff
@@ -198,22 +198,28 @@ func doTestRegistryClientImage(t *testing.T, registry, version string) {
 
 	var image *dockerregistry.Image
 	err = retryWhenUnreachable(t, func() error {
-		image, err = conn.ImageByTag("openshift", "origin", "latest")
+		image, err = conn.ImageByTag("openshift", reponame, "latest")
 		return err
 	})
 	if err != nil {
 		t.Fatal(err)
 	}
-	if len(image.ContainerConfig.Entrypoint) == 0 {
-		t.Errorf("%s: unexpected image: %#v", version, image)
+
+	if image.Comment != "Imported from -" {
+		t.Errorf("%s: unexpected image comment", version)
+	}
+
+	if image.Architecture != "amd64" {
+		t.Errorf("%s: unexpected image architecture", version)
```

Comment:
This is done in order to check the filling metadata. In other words what config blob was obtained. This is a test repository so that I can be sure that they need to be there.